stylelint-polaris/scripts/gen-polaris-data.js

```diff
@@ -1,10 +1,8 @@
 const fs = require('fs');
 const path = require('path');
 
-const {
-  getCustomPropertyNames,
-} = require('../../polaris-react/scripts/utilities/getCustomPropertyNames');
-const getKeyframeNames = require('../../polaris-react/scripts/utilities/getKeyframeNames');
+const getCustomPropertyNames = require('../../polaris-tokens/scripts/getCustomPropertyNames');
+const getKeyframeNames = require('../../polaris-tokens/scripts/getKeyframeNames');
```

Comment:
`@shopify/polaris-tokens` should really be a dependency in `stylelint-polaris` e.g.
```js
const {getCustomPropertyNames, getKeyframeNames} = require('@shopify/polaris-tokens')
```

Also, please remove this `gen-polaris-data` script entirely, as it was only a temporary solution until this package had access to `@shopify/polaris-tokens`